### PR TITLE
Twitter wipe improved to support json imports from Twitter archive

### PR DESCRIPTION
**Pull request by dantebarba**
As you may already know, twitter has decided to limit tweet retrieval
to 3200 tweets.

To circumvent this limitation, I've implemented the possibility to import the JSON file provided by Twitter archive, to import all the tweets and likes ids, thus allowing to delete every single one by id. Furthermore, it's seems that we are in luck, because there is no limitation on how many tweets you can process by id. This allows you to delete all your tweets in one pass.

The application has been tested with an account with over 15k tweets.
It deleted not just the tweets but retweets and likes/fav

Plus, a CLI has been added to facilitate configuration. You can access the help
menu using --help

And just to add a cherry on top, it has been fully dockerized, and it includes
a docker-compose.yml example.

TLDR;

- Delete tweets by id using json
- Delete favs by id using json
- CLI, using "click" library
- Dockerized, with docker-compose as example.

Roadmap?

- Maybe CI? Travis?
- Improve logging. When API Cursor or Auth fails, there is no log information.
- Improve logging, give time-remaining estimates or log every n-th processed
entry, to keep track of what the
script is doing
- Improve summary.

### CODE REVIEW

**Comment by murdoc02** [comment]
hey! i looked over this, and i'm not sure containerizing the script makes this easier to use. i'd like to keep this a simple script, though possibly add some sort of UI at some point.

**Comment by dantebarba** [comment]
I see no harm on including a Dockerfile in it. If the user doesnt want to run it with Docker they could do it anyways...
